src/sage/combinat/finite_state_machine_generators.py

```diff
@@ -1989,8 +1989,8 @@ def f(n):
                     cycle[1:])
             required_initial_values.update(intersection)
             output_sum = sum([edge[2]
-                              for edge in recursion_digraph.\
-                                  outgoing_edge_iterator(cycle[1:])],
+                              for edge in recursion_digraph.
+                              outgoing_edge_iterator(cycle[1:])],
```

Comment:
It looks really bad that the attribute after dot starts in a new line. It seems better here to break PEP8.